Documentation/Authentication.md

```diff
@@ -1,140 +1,119 @@
+# Authentication
```

Comment:
The Doc had link to a script that helps you create an sp in node.js. If someone did not want to use the portal or the CLI, they can use this script to create an sp, assign a role to it and then login as that sp.
The script can be found [here](https://github.com/Azure/azure-sdk-for-node/tree/master/Documentation/ServicePrincipal). Please provide reference to this script in the walk through